repo_nginx_fancyindex/footer_en.html

```diff
@@ -39,19 +31,19 @@ <h2><span>Post</span>-Installation</h2>
 
       <p>Once your Cube found, connect with SSH on port 22:</p>
 
-      <pre>root:olinux</pre>
+      <pre>root:yunohost</pre>
```

Comment:
admin

Reply:
Hm nope, not before the post-install has been ran ... but granted that this can bring confusion :/ Yet root login is always possible on local-network though